### DISCUSSION TITLE
GitHub Student Account Activation Issue

### DISCUSSION THREAD

**Discussion by 3mmarYasser** [community/community#119693] [GitHub Education] [answered]
### Hi everyone,

Dear GitHub Support Team,

I recently applied for the GitHub Student Developer Pack, which was approved, but my student account hasn't been activated after four days. Despite receiving a message to wait for activation, I'm still unable to access the benefits. Could you please investigate and assist with activating my account promptly? .Thank you for your prompt attention to this matter.

**Comment by moozunch** [comment, marked as answer]
it says it approved only means your application is being checked, if you are accepted you will get a congratulation email and you're account will automatically turn into a pro account, you will need wait a bit more till they are finish checking your request (usually take few days, mine 4 days but it can be faster or slower depends on the situation), don't worry I believed you will get your benefit soon ^_____^

**Reply by 3mmarYasser** [reply]
Got it, thanks! Just to update you, my account has actually been activated already. Thanks again for your assistance!

**Comment by goldn-apk** [comment]
Dear GitHub Support Team,

I recently applied for the GitHub Student Developer Pack, which was approved, but my student account hasn't been activated after four days. Despite receiving a message to wait for activation, I'm still unable to access the benefits. Could you please investigate and assist with activating my account promptly? .Thank you for your prompt attention to this matter.

**Comment by RohitRajAstro** [comment]
Dear Github support team, I have applied for Github student developer pack/resources. I have my college email attached to my account. I tried to put different sorts of verification docs but after clicking the process button, nothing happens. Im not sure if my application even went through. Help me please.

**Comment by mohamediflal** [comment]
Dear developer team. I applied to educational benefits. but got an issue with verification of my name. tried several times using my university ID card name but still showing as name is not match. please consider this issue and reply to me soon

**Comment by divanshmittal** [comment]
Dear Github support team, my student account has been approved but i still dont have access to the benefits even after 4 days.

**Comment by guransharora** [comment]
Please approve my account. The application was approved 3 days ago and ive yet to be activated
